README.md

```diff
@@ -47,13 +47,20 @@ You can also use this helm chart to deploy the metric-server in your cluster (Th
 
 ## FAQ
 
+#### What metrics are exposed by metrics server?
+
+Metrics server collects resource usage metrics needed for autoscaling: CPU & Memory.
+Metric values use standard kubernetes units (`m`, `Ki`), same as those used to
+define pod requests and limits (Read more [Meaning of CPU](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu), [Meaning of memory](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory))
+Metrics server itself is not responsible for calculating metric values, this is done by Kubelet.
+
 #### When metrics server is released?
 
 There is no hard release schedule. Release is done after important feature is implemented or upon request.
 
 #### Can I run two instances of metrics-server?
 
-Yes, api requests will be loadbalanced between them, but both will be scraping all nodes.
+Yes, but you will provide any benefits. All Metrics will be scraped by both instances, but only one will be serving metrics.
```

Comment:
"but you will provide any benefits" - this looks like a mistake, did you mean: "but you won't get any benefits"?